contrib/layout.libsonnet

```diff
@@ -0,0 +1,58 @@
+// layout.libsonnet - a little mixin for the Grafana dashboard type that
+// automatically arranges panels on a dashboard, so you don't have to manually
+// manage the positions anymore.
+//
+// We track a cursor in the dashboard and add panels at the cursor, moving
+// the cursor to the right after each new panel.  We also expose a function
+// to move the cursor to the next row, and track the height of the panels added
+// to make this possible.
+//
+// Usage
+//   local grafana = import 'grafonnet-7.0/grafana.libsonnet';
+//   local layout = import "grafonnet-7.0/layout.libsonnet";
```

Comment:
After the move to `contrib`, this example doesn't work out of the box any more.

Shouldn't `contrib` be inside `grafonnet-7.0`?

As in: `local layout = import 'grafonnet-7.0/contrib/layout.libsonnet';`